pkg/cluster/root_app.go

```diff
@@ -60,8 +63,15 @@ func ConstructRootApp(
 		app.ObjectMeta.Labels["arlon-cluster"] = clusterName
 		// assume CAPI for now
 		apiProvider = "capi"
+		if withCAS {
+			err := enableClusterAutoscaler(apiProvider, &helmParams, &ignoreDiffs)
+			if err != nil {
+				return nil, err
+			}
+			fmt.Println(helmParams, ignoreDiffs)
+		}
 	}
-	helmParams := []argoappv1.HelmParameter{
+	helmParams = append(helmParams, []argoappv1.HelmParameter{
```

Comment:
Wouldn't it be more elegant to:
```
helmParams = append(helmParams,
{
 Name: xxx,
 Value: yyy
},
...
)
```
instead of using ellipsis to expand the array